chapters/11-version-control-workflow.Rmd

```diff
@@ -5,25 +5,30 @@
 - E.g. with pr_* functions from usethis
```

Comment:
I think its best to keep it narrow and focused. I'd prefer to keep is simple by focusing on the interaction between usethis and using PRs on GitHub. Just because I feel the pr_helpers are soooo helpful!